tests/unit/api/test_offline_ingest_query_graph.py

```diff
@@ -937,6 +944,16 @@ def test_databricks_specs(
             },
             output_name="req_col_feature",
         ),
+        FeatureLookup(
+            table_name="feature_engineering.some_schema.cat1__no_entity_30m",
+            lookup_key=["__featurebyte_dummy_entity"],
+            timestamp_lookup_key=timestamp_lookup_key,
+            lookback_window=None,
+            feature_names=["latest_event_timestamp_overall_90d_V240103"],
+            rename_outputs={
+                "latest_event_timestamp_overall_90d_V240103": "latest_event_timestamp_overall_90d"
+            },
+        ),
     ]
```

Comment:
Revise feature specs ordering so that it is aligned with original feature list's feature ordering.